client/bindings/java/lib/src/test/java/org/iota/IndexerApiTest.java

```diff
@@ -3,53 +3,133 @@
+                .withAliasId(new AliasId("0x0000000000000000000000000000000000000000000000000000000000000000"))
+                .withUnlockConditions(new UnlockCondition[]{
+                        new UnlockCondition("{ type: 4, address: { type: 0, pubKeyHash: \"" + hexAddress + "\" } }"),
+                        new UnlockCondition("{ type: 5, address: { type: 0, pubKeyHash: \"" + hexAddress + "\" } }")
+                });
+
+        Map.Entry<BlockId, Block> response = client.buildAndPostBlock(s, new BuildBlockOptions().withOutputs(new Output[] { client.buildAliasOutput(p) }));
+        Thread.sleep(1000 * 25);
+        TransactionId transactionId = client.getTransactionId(new TransactionPayload(response.getValue().toJson().get("payload").getAsJsonObject()));
+
+        // Build the Foundry Output
+        client.requestTestFundsFromFaucet(address);
+        AliasId aliasId = client.computeAliasId(new OutputId(transactionId + "0000" ));
+        int serialNumber = 1;
+        TokenScheme tokenScheme = new TokenScheme("{ type: 0, meltedTokens: '0x0', mintedTokens: '0x32', maximumSupply: '0x64' }");
+        UnlockCondition[] unlockConditions = new UnlockCondition[]{new UnlockCondition("{ type: 6, address: { type: 8, aliasId: " + aliasId + "  } }")};
+        FoundryOutputBuilderParams params = new FoundryOutputBuilderParams()
+                .withSerialNumber(serialNumber)
+                .withTokenScheme(tokenScheme)
+                .withUnlockConditions(unlockConditions);
+        Output foundryOutput = client.buildFoundryOutput(params);
+
+        // Build the new foundry output
+        Output newAliasOutput = client.buildAliasOutput(new AliasOutputBuilderParams()
+                .withAliasId(aliasId)
+                .withFoundryCounter(1)
+                .withStateIndex(1)
+                .withUnlockConditions(new UnlockCondition[]{
+                        new UnlockCondition("{ type: 4, address: { type: 0, pubKeyHash: \"" + hexAddress + "\" } }"),
+                        new UnlockCondition("{ type: 5, address: { type: 0, pubKeyHash: \"" + hexAddress + "\" } }")
+                }));
+
+        // Create the transaction and use the outputs
+        client.buildAndPostBlock(s, new BuildBlockOptions()
+                .withOutputs(new Output[] {
+                        foundryOutput,
+                        newAliasOutput
+                })
         );
-        Output o = new Output(json);
 
-        client.buildAndPostBlock(new MnemonicSecretManager(DEFAULT_DEVELOPMENT_MNEMONIC), new BuildBlockOptions().withOutputs(new Output[] { o }));
+        Thread.sleep(1000 * 25);
 
-        for (OutputId outputId : client.getFoundryOutputIds(new NodeIndexerApi.QueryParams().withParam("aliasAddress", "rms1pz022kd0zjmu4dms8whgnfus3ph347cmr8th3kk5g2qn7sl5xzfcj33gfu4")))
+        for (OutputId outputId : client.getFoundryOutputIds(new NodeIndexerApi.QueryParams().withParam("aliasAddress", client.aliasIdToBech32(aliasId, "rms"))))
```

Comment:
Might be good to get the bech32 HRP from the client, in case we want to change the node in the future (for example private tangle where it's `tst` then)